spec/models/user_spec.rb

```diff
@@ -587,7 +587,10 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
   end
 
   it "inserts into mailchimp" do
-    expect(user.subscribe_to_mailchimp_newsletter_without_delay).to eq true
+    perform_enqueued_jobs do
+      expect_any_instance_of(MailchimpBot).to receive(:upsert)
```

Comment:
Let's not use `expect_any_instance_of` when it's not absolutely necessary, and rubocop should have complained about it.
What I suggested was:
- prepare a `user`
- call the method `subscribe_to_mailchimp_newsletter`
- check that the job was enqueued with the help of the `have_enqueued_job` helper

Reply:
done